src/core/Akka.Streams.Tests.Performance/FlowSelectBenchmark.cs

```diff
@@ -54,7 +54,7 @@ public class FlowSelectBenchmark
         public void Setup(BenchmarkContext context)
         {
             _actorSystem = ActorSystem.Create("FlowSelectBenchmark", Config.WithFallback(
-                ConfigurationFactory.FromResource<ScriptedTest>("Akka.Streams.TestKit.Tests.reference.conf")));
```

Comment:
The magic text for loading the config is moved to a static property in the `StreamTestDefaultMailbox` class